src/processors/default/processPlatformTracks/processNFTFactoryTracks.ts

```diff
@@ -83,14 +83,15 @@ const createTracks = async (
 }
 
 export const getTrackInputs = async (
-  mapNFTsToTrackIds: (nfts: NFT[], dbClient?: DBClient | undefined, apiTracksByNFT?: any) => Promise<{
+  mapNFTsToTrackIds: (nfts: NFT[], dbClient?: DBClient | undefined, titleExtractor?: any, apiTracksByNFT?: any) => Promise<{
     [trackId: string]: NFT[];
   }>,
   nfts: NFT[],
   dbClient: DBClient,
+  titleExtractor: any,
   apiTracksByNFT: any
   ) => {
-  const trackMapping = await mapNFTsToTrackIds(nfts, dbClient, apiTracksByNFT);
+  const trackMapping = await mapNFTsToTrackIds(nfts, dbClient, titleExtractor, apiTracksByNFT);
```

Comment:
you've changed the order of parameters being sent here. there may be (and i think there are) mappers that expect apiTracksByNFT to be the third argument in this function call, so this may break other stuff. I think just put it at the end, or alternatively we can do a bigger refactor to the mapNFTsToTrackIds function that accepts an object as an argument so that it's not order-dependent

Reply:
Yeah, we have a lot of different type annotations for `mapNFTsToTrackIds` across the codebase and I had assumed the more specific ones would override any others. But you're right, this isn't a solid choice right now. I've adjusted the order 👍